src/codegen/templates/library_interpreter/default_c_function_call.py.mako

```diff
@@ -1,12 +1,12 @@
 <%page args="function"/>\
 <%
-    from codegen.utilities.interpreter_helpers import generate_interpreter_function_call_args, get_samps_per_chan_read_or_write_param, get_argument_types
+    from codegen.utilities.interpreter_helpers import generate_interpreter_function_call_args, get_samps_per_chan_read_or_write_param, get_argument_types, is_attribute_function
     from codegen.utilities.text_wrappers import wrap, docstring_wrap
 %>\
         cfunc = lib_importer.${'windll' if function.calling_convention == 'StdCall' else 'cdll'}.DAQmx${function.c_function_name}
 \
 ## Create argument ctypes types list.
-%if function.calling_convention == 'StdCall':
+%if function.calling_convention == 'StdCall' or is_attribute_function(function):
```

Comment:
Now there are only two functions that use the else case:
- `get_analog_power_up_states_with_output_type`
- `set_analog_power_up_states_with_output_type`

(How do I know? I edited the template and added a comment to the else case.)

These functions should also use double-checked locking. Delete the if-statement and its else case.

Reply:
I have removed the if/else blocks.